views/_includes/radiosMixin.pug

```diff
@@ -10,9 +10,15 @@ mixin radios(id, legend, value, options)
         if errors && errors[id]
           +validationMessage(errors[id].msg, id)
         each opt, index in options
-          .multiple-choice__item
-            input(id=(id+index) name=id type="radio" value=opt.value aria-describedby=(errors && errors[id] ? id+"-error" : false) checked=(value === opt.value))
-            label(for=(id+index)) #{__(opt.label)}
+          if opt.label === "Skip security questions"
+            p.multiple-choice__p #{__('OR')}
+            .multiple-choice__item
+              input(id=(id+index) name=id type="radio" value=opt.value aria-describedby=(errors && errors[id] ? id+"-error" : false) checked=(value === opt.value))
+              label(for=(id+index)) #{__(opt.label)}
+          else
+            .multiple-choice__item
+              input(id=(id+index) name=id type="radio" value=opt.value aria-describedby=(errors && errors[id] ? id+"-error" : false) checked=(value === opt.value))
+              label(for=(id+index)) #{__(opt.label)}
```

Comment:
Pretty sure you can un-nest the multiple-choice__item stuff.

ie, 
```js
          if opt.label === "Skip security questions" 
            // this will only appear on the conditional
            p.multiple-choice__p #{__('OR')}

          // this will appear everytime
          .multiple-choice__item
            input(id=(id+index) name=id type="radio" value=opt.value aria-describedby=(errors && errors[id] ? id+"-error" : false) checked=(value === opt.value))
            label(for=(id+index)) #{__(opt.label)}
```